### PR TITLE
Update to Wicket 10 / Jakarta EE 10

### PR DESCRIPTION
**Pull request by renoth**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -60,12 +60,12 @@
...
 		<maven.compiler.source>11</maven.compiler.source>
 
-		<wicket.version>9.16.0</wicket.version>
+		<wicket.version>10.0.0-M2</wicket.version>
 		<jackson.version>2.16.1</jackson.version>
 		<slf4j.version>2.0.12</slf4j.version>
 		<junit.version>5.10.2</junit.version>
-		<servlet-api.version>4.0.4</servlet-api.version>
-		<jetty.version>9.4.54.v20240208</jetty.version>
+		<servlet-api.version>6.0.0</servlet-api.version>
```
Comment (martin-g) [inline pom.xml:67]: Wicket uses Servlet API 5.x. Do we require 6.x for Wiquery ?

Comment (renoth) [reply pom.xml:67]: ah no, i will go to 5 instead

**Comment by martin-g** [comment]
Please also update the Github Action workflow to use Java 17

**Comment by martin-g** [comment]
Thank you, @renoth !

**Pull request merged**